pandas/core/algorithms.py

```diff
@@ -717,6 +717,17 @@ def select_n(series, n, keep, method):
     return dropped.iloc[inds]
 
 
+def select_n_frame(frame, columns, n, method, keep):
+    from pandas.core.series import Series
```

Comment:
can you add doc-strings to ``select_n_*`` methods?. otherwise lgtm. ping on green.